tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Note that this tutorial requires astropy v1.0 or higher."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "import urllib\n",
+      "import IPython.display\n",
+      "import numpy as np"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%matplotlib inline\n",
+      "from matplotlib import pyplot as plt"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astropy import units as u\n",
+      "from astropy.coordinates import SkyCoord\n",
+      "from astropy.table import Table"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "heading",
+     "level": 2,
+     "metadata": {},
+     "source": [
+      "Usinging Coordinates to describe on-sky locations and match catalogs"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Lets start by considering a field around the picturesque Hickson Compact Group 7.  To do anything with this, we need to get an object that represents the coordinates of the center of this group.\n",
+      "\n",
+      "In Astropy, the most common object you'll work with for coordinates is `SkyCoord`.  A `SkyCoord` can be created most easily directly from angles as shown below.  It's also wise to explicitly specify the frame your coordinates are in, although this is not strictly necessary because the default is ICRS. \n",
+      "\n",
+      "(If you're not sure what ICRS is, it's basically safe to think of it as an approximation to an equatorial system at the J2000 equinox)."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord(9.81625*u.deg, 0.88806*u.deg, frame='icrs')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "SkyCoord will also accept string-formatted coordinates, although you'll need to give units if it's not obvious from the format."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0h39m15.9s', '0d53m17.016s', frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0:39:15.9', '0:53:17.016', unit=(u.hour, u.deg), frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
```

Comment:
1) yes, that does work, but its discouraged because it's harder to read (requires an extra keyword and its less immediately obvious which goes with which if its something like ``(u.hour, u.deg)`` - the string example with ``unit`` is there because sometimes strings you get from elsewhere are ambiguous).  So that's why it's not shown as an example.

2) Indeed, ``unit=u.deg`` makes them both degrees.  There was some debate about this question (I actually preferred ``units`` originally), but the conclusion was that it's better to have unit keywords *always* be ``unit`` so that you don't have to wonder which it might be for `SkyCoord` vs `Quantity` or `Distance` or whatever.

3) Yes, that works just like you'd expect.  I'd prefer not to demonstrate that here, though, because it's formally "discouraged", even though it's practically useful.

I did make some changes to the wording prompted by your questions here, though, @embray.  So this may get hidden if git concludes this particular line was edited...